README.md

```diff
@@ -123,6 +123,7 @@ List of options provided by CMake:
 | USE_TSAN | Enable ThreadSanitizer checks | ON/OFF | OFF |
 | USE_MSAN | Enable MemorySanitizer checks | ON/OFF | OFF |
 | USE_VALGRIND | Enable Valgrind instrumentation | ON/OFF | OFF |
+| USE_GCOV | Enable gcov support | ON/OFF | OFF |
```

Comment:
please add a note somewhere that this will work only for linux

Reply:
Done